web/src/main/webapp/swagger-ui/lib/backbone-min.js

```diff
@@ -19,16 +19,1119 @@
+(function(t, e) {
```

Comment:
.min filer borde vi nog inte ta med i formatteringen? Nu vet jag dock inte om vi ens har stöd för just swagger längre, eller om vi i övriga appar checkat in .min.js filer - men kan vara värt att kolla?